vcf-dumper/vcf-dumper-ws/src/main/java/uk/ac/ebi/eva/vcfdump/server/configuration/SwaggerParameterDescriptions.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2020 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.vcfdump.server.configuration;
+
+public class SwaggerParameterDescriptions {
+
+    public static final String STUDY_DESCRIPTION = "Study identifier (separate with comma for multiple studies), " +
+            "e.g. PRJEB9799. Individual study identifiers can be looked up in " +
+            "https://www.ebi.ac.uk/eva/webservices/rest/v1/meta/studies/all in the field named 'id'.";
+
+    public static final String STUDY_LIST_DESCRIPTION = "Study identifiers, e.g. PRJEB9799. Each individual " +
```

Comment:
Looks like both this and the description above are meant for list of study identifiers?

Reply:
That separation was because the endpoint of the vcf dumper and the htsget has different data types for the species (List<String> and String) so they had different descriptions. I will change it so both are Strings and the description can be reused

Reply:
Agreed offline to keep it as it is as changing the structure could break if there are users using it as it is now. Updated the swagger documentation to make it clearer.